tests/conftest.py

```diff
@@ -0,0 +1,492 @@
+# pylint: disable=redefined-outer-name
+
+import pytest
+from graph_structures_pb2 import (
+    SLI,
+    Client,
+    Dependency,
+    Node,
+    NodeType,
+    Status,
+    UserJourney)
+
+import ujt.constants
+
+
+@pytest.fixture
+def assert_same_elements():
+
+    def inner_assert_same_elements(list1, list2):
+        """ Asserts that two lists have the same elements, regardless of order.
+
+        We use this approach for unhashable and uncomparable types, e.g. proto messages.
+
+        Args:
+            list1: First list to assert.
+            list2: Second list to assert.
+        """
+        # although it's slower/not strictly necessary to wrap the generator in
+        # a list, pytest will give a more informative error message this way.
+        assert all([element1 in list2 for element1 in list1])
+        assert all([element2 in list1 for element2 in list2])
+
+    return inner_assert_same_elements
+
+
+@pytest.fixture
+def slo_bounds():
+    return {
+        "slo_error_lower_bound": .1,
+        "slo_warn_lower_bound": .2,
+        "slo_warn_upper_bound": .8,
+        "slo_error_upper_bound": .9,
+    }
+
+
+""" Example Node name message map data:
+Service0 has children Endpoint0 and Endpoint1
+Service1 has child Endpoint2
+
+Endpoint0 depends on Endpoint1 and Endpoint2
+Endpoint1 depends on Endpoint2
+"""
+
+
+@pytest.fixture
+def example_node_name_message_map_service_relative_names():
+    service_relative_names = ["Service0", "Service1"]
+    return service_relative_names
+
+
+@pytest.fixture
+def example_node_name_message_map_endpoint_relative_names():
+    endpoint_relative_names = ["Endpoint0", "Endpoint1", "Endpoint2"]
+    return endpoint_relative_names
+
+
+@pytest.fixture
+def example_node_name_message_map(
+    slo_bounds,
+    example_node_name_message_map_service_relative_names,
+    example_node_name_message_map_endpoint_relative_names,
+):
```

Comment:
Refactored some test input into these fixtures now that multiple tests use them.